procedures/ManageCertificatesAndRequests/form.xml

```diff
@@ -0,0 +1,99 @@
+        <option><name>Clean</name><value>clean</value></option>
+        <option><name>Finger Print</name><value>fingerprint</value></option>
+        <option><name>Generate</name><value>generate</value></option>
+        <option><name>List</name><value>list</value></option>
+        <option><name>Print</name><value>print</value></option>
+        <option><name>Revoke</name><value>revoke</value></option>
+        <option><name>Sign</name><value>sign</value></option>
+        <option><name>Verify</name><value>verify</value></option>
+        <option><name>Reinventory</name><value>reinventory</value></option>
+    </formElement>
+    <formElement>
+        <type>checkbox</type>
+        <label>All:</label>
+        <property>all</property>
+        <required>0</required>
+        <documentation>Operate on all items. Currently only makes sense with the 'sign', 'clean', 'list', and 'fingerprint' actions.</documentation>
+        <checkedValue>1</checkedValue>
+        <uncheckedValue>0</uncheckedValue>
+    </formElement>
+    <formElement>
+        <type>checkbox</type>
+        <label>Digest:</label>
+        <property>digest</property>
+        <required>0</required>
+        <documentation>Set the digest for fingerprinting (defaults to the digest used when signing the cert). Valid values depends on your openssl and openssl ruby extension version.</documentation>
+        <checkedValue>1</checkedValue>
+        <uncheckedValue>0</uncheckedValue>
+    </formElement>
+    <formElement>
+        <type>checkbox</type>
+        <label>Debug:</label>
+        <property>debug</property>
+        <required>0</required>
+        <documentation>Enable full debugging.</documentation>
+        <checkedValue>1</checkedValue>
+        <uncheckedValue>0</uncheckedValue>
+    </formElement>
+    <formElement>
+        <type>checkbox</type>
+        <label>Help:</label>
+        <property>help</property>
+        <required>0</required>
+        <documentation>Print this help message.</documentation>
+        <checkedValue>1</checkedValue>
+        <uncheckedValue>0</uncheckedValue>
+    </formElement>
+    <formElement>
+        <type>checkbox</type>
+        <label>Verbose:</label>
+        <property>verbose</property>
+        <required>0</required>
+        <documentation>Enable verbosity.</documentation>
+        <checkedValue>1</checkedValue>
+        <uncheckedValue>0</uncheckedValue>
+    </formElement>
+    <formElement>
+        <type>checkbox</type>
+        <label>Version:</label>
+        <property>version</property>
+        <required>0</required>
+        <documentation>Print the puppet version number and exit.</documentation>
```

Comment:
Version option causes the command to print the version number and exit . The actual command does not run along with the version option.So this will be same as providing a separate procedure for the same . Do you still want a separate procedure to be implemented for the same?